src/data/decimal/decimalFn.spec.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  assertNumeric,
+  isScientific,
+  plDecimal,
+  simplifyDecimal,
+  parseDecimalString,
+  parseScientificString,
+  parseNumString,
+  expandDecimals,
+  getDecimalString,
+} from './decimalFn'
+import { PLDecimal } from './PLDecimal'
+
+const pld = plDecimal
+
+describe('stdlib/data/decimal/decimalFn', () => {
+  describe('plDecimal', () => {
+    it('should construct decimal', () => {
+      expect(pld('0.2')).toEqual(new PLDecimal(2, 1))
```

Comment:
Testing is like playing chess with yourself, you must give your best to beat your implementation in the tests cases.
So you need to think about edge cases especially when you are doing string parsing where the input data set is much bigger then with numbers.

So please add some edge cases like `pld('hello world')` or `pld('0.2hello')` etc.
Because all our functions use the same parser you can do edge cases testings ones

Reply:
i added some checks for plNumber to handle edge cases